apps/api/src/database/migrations/2692870736644-migration.ts

```diff
@@ -0,0 +1,237 @@
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_users',
+        columns: [
+          {
+            name: 'user_id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            isUnique: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'username',
+            type: 'varchar',
+          },
+          {
+            name: 'password',
+            type: 'varchar',
+          },
+          {
+            name: 'role',
+            type: 'tinyint',
+            default: 0,
+          },
+          {
+            name: 'application_id',
+            type: 'int',
+          },
+          {
+            name: 'created_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_master_providers',
+        columns: [
+          {
+            name: 'provider_id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            isUnique: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+          },
+          {
+            name: 'type',
+            type: 'enum',
+            enum: ['sms', 'email', 'whatsapp', 'others'],
+            default: "'sms'",
+          },
+          {
+            name: 'configuration',
+            type: 'text',
+          },
+          {
+            name: 'created_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+
+    // Add a column in notify_notifications
+    await queryRunner.addColumn(
+      'notify_notifications',
+      new TableColumn({
+        name: 'application_id',
+        type: 'int',
+        // Kept as null for now
+        isNullable: true,
+        default: null
+      }),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Add another query which drops the `application_id` column we add to `notify_notifications` when running this migration.